tests/prost-path/src/lib.rs

```diff
@@ -0,0 +1 @@
+include!(concat!(env!("OUT_DIR"), "/prost_path.rs"));
```

Comment:
This test doesn't actually test anything. Please verify the correct types are used and explain what this test is testing.

Reply:
The dependencies in `Cargo.toml` only include the `reexported_prost` crate, and the configuration in `build.rs` contains the following:

```rust
fn main() {
    prost_build::Config::new()
        .prost_path("::reexported_prost::prost")
        .prost_types_path("::reexported_prost::prost_types")
        .compile_protos(&["protos/prost_path.proto"], &["protos"])
        .unwrap();
}
```

Therefore if it successfully compiles, it means that the change is working as expected.

Is there something I'm missing here, or would it be enough to just add that as a comment above?